src/tiled/mainwindow.cpp

```diff
@@ -316,8 +316,10 @@ MainWindow::MainWindow(QWidget *parent, Qt::WindowFlags flags)
     connect(mUi->actionZoomNormal, SIGNAL(triggered()), SLOT(zoomNormal()));
     connect(mUi->actionFullScreen, &QAction::toggled, this, &MainWindow::setFullScreen);
 
+    CommandManager::instance()->registerMenu(mUi->menuCommand);
+
     connect(mUi->menuCommand, &QMenu::aboutToShow, 
-            [this]() { CommandManager::instance()->populateMenu(mUi->menuCommand); });
+            [this]() { CommandManager::instance()->populateMenu(); });
```

Comment:
You don't need this connection if you're keeping the menu contents updated already anyway.